### ISSUE TITLE
Unable to build KDoc under JDK9

### ISSUE THREAD

**Issue by simonbasle** [Kotlin/dokka#220]
When trying to build the KDoc using dokka and the gradle plugin, I get the following error:

```
dokka
Download http://repo.spring.io/plugins-release/org/jetbrains/dokka/dokka-fatjar/0.9.15/dokka-fatjar-0.9.15.pom
Download http://repo.spring.io/plugins-release/org/jetbrains/dokka/dokka-fatjar/0.9.15/dokka-fatjar-0.9.15.jar
:dokka FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':dokka'.
> Unable to make field private static final jdk.internal.misc.Unsafe jdk.internal.misc.Unsafe.theUnsafe accessible: module java.base does not "opens jdk.internal.misc" to unnamed module @5dd3a459
```

As visible in the logs I'm using dokka `0.9.15`. I'm also using the `kotlin-gradle-plugin` v`1.1.51`.

Note that I also configured the dokka task in a way to work around #217

**Comment by lehvolk** [comment]
Any plans to fix it?

**Comment by semoro** [comment]
It already fixed in 0.9.16 EAP

https://github.com/Kotlin/dokka/releases/tag/0.9.16-eap-3

**Comment by lehvolk** [comment]
@semoro thanks.

**Comment by semoro** [comment]
Fixed in [0.9.16](https://github.com/Kotlin/dokka/releases/tag/0.9.16)
